src/Runner.Sdk/Util/UrlUtil.cs

```diff
@@ -8,7 +8,8 @@ public static bool IsHostedServer(UriBuilder gitHubUrl)
         {
             return string.Equals(gitHubUrl.Host, "github.com", StringComparison.OrdinalIgnoreCase) ||
                 string.Equals(gitHubUrl.Host, "www.github.com", StringComparison.OrdinalIgnoreCase) ||
-                string.Equals(gitHubUrl.Host, "github.localhost", StringComparison.OrdinalIgnoreCase);
+                string.Equals(gitHubUrl.Host, "github.localhost", StringComparison.OrdinalIgnoreCase) ||
+                gitHubUrl.Host.EndsWith("ghe.com", StringComparison.OrdinalIgnoreCase);
```

Comment:
`EndsWith("ghe.com"` or `EndsWith(".ghe.com"`?

Reply:
Good point. I'll fix that.

Reply:
Would GHES ever be on `ghe.com`?

Reply:
You never know what customers will try to use in their network.